pom.xml

```diff
@@ -84,6 +84,7 @@
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-compat</artifactId>
             <version>${maven3.version}</version>
+            <scope>provided</scope>
```

Comment:
Unrelated, but `org.apache.maven` plugins should be `provided` (Maven warning message).

Reply:
"Except" that some artifacts actually aren't provided. Warning is too aggressive. I've already done as much clean up as possible.